### PR TITLE
Unit and integration tests

### PR DESCRIPTION
**Pull request by wtchnm**

### CODE REVIEW

File: src/components/__tests__/Fruit.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+jest.mock('react-router-dom', () => ({
+	...jest.requireActual('react-router-dom'),
+	useHistory: (): Partial<ReturnType<typeof useHistory>> => ({
+		push: mockHistoryPush
+	})
+}))
+
+describe('<Fruit />', () => {
+	beforeEach(() => {
+		render(<Fruit fruit={fruits[0]} index={0} />)
```
[the remark this answers is not in the document]
Comment (wtchnm) [reply src/components/__tests__/Fruit.tsx:17]: Yeah, I saw that they made this rule recommended in the last update.
Just out of curiosity, do you have any idea why this is forbidden? It is not mentioned in the documentation.

[the remark this answers is not in the document]
Comment (wtchnm) [reply src/components/__tests__/Fruit.tsx:17]: With your comment I realized that the lint command is not considering test files. Thanks!

**Review by nonameolsson — APPROVED** [review]

**Comment by nonameolsson** [comment]
For me it is OK to merge this one @wtchnm. Just the conflicts with main to solve.

**Comment by wtchnm** [comment]
> For me it is OK to merge this one @wtchnm. Just the conflicts with main to solve.

@nonameolsson merged! Thanks for the help!

**Pull request merged**